final_task/tests/test_console_interface.py

```diff
@@ -0,0 +1,28 @@
+import sys
+sys.path.insert(1, 'final_task/rss_reader')
```

Comment:
`sys.path.append` will work here too, there is no need to insert it to the second position of the list.